server/src/test/java/org/elasticsearch/index/query/RandomQueryBuilder.java

```diff
@@ -47,7 +48,7 @@ public static QueryBuilder createQuery(Random r) {
             case 1:
                 return new TermQueryBuilderTests().createTestQueryBuilder();
             case 2:
-                return new IdsQueryBuilderTests().createTestQueryBuilder();
+                return new ExistsQueryBuilder(randomAlphaOfLength(5));
```

Comment:
Was this changed so we don't get warnings all over the place? If so, how about nulling the "type" after creating the random query builder? I think it would be nice to keep this the same type of query builder as before, just to not have to many changes in the tests at once that might be introduced by changing this to "exists", although that might be paranoid.

Reply:
Sounds good, I will change it back to an `ids` query with a comment as to why we're nulling out the types.